swedbank-pay-woocommerce-payments.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /*
- * Plugin Name: PayEx WooCommerce Payments
- * Plugin URI: http://payex.com/
- * Description: Provides a Credit Card Payment Gateway through PayEx for WooCommerce.
- * Author: PayEx
- * Author URI: http://payex.com/
+ * Plugin Name: Swedbank Pay WooCommerce Payments
+ * Plugin URI: https://www.swedbankpay.se/
+ * Description: Provides a Credit Card Payment Gateway through Swedbank Pay for WooCommerce.
+ * Author: Swedbank Pay
+ * Author URI: https://www.swedbankpay.se/
  * License: Apache License 2.0
  * License URI: http://www.apache.org/licenses/LICENSE-2.0
  * Version: 1.3.1
- * Text Domain: payex-woocommerce-payments
+ * Text Domain: sb-woocommerce-payments
```

Comment:
Why the abbreviation of `swedbank-pay` to `sb`?

```suggestion
 * Text Domain: swedbank-pay-woocommerce-payments
```